### PR TITLE
Bayes model: add model evaluation scripts, readme

### PR DESCRIPTION
**Pull request by wangfan860**
Jira Ticket: [COV-1075](https://occ-data.atlassian.net/browse/COV-1075)

### New Features
* Bayes model: add readme files for model evaluation scripts and generative model scripts.
* Bayes model: add  model evaluation scripts

### CODE REVIEW

**Review by kmhernan — DISMISSED** [review]

File: generative_bayes_model/readme.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+* Model time-varying processes.
+
+* Model Infection delay (generation time).
+
+
+
+## Model Design
+
+1. Primary infection: <img src="https://render.githubusercontent.com/render/math?math=y_t%20=%20y_{t-1}%20\cdot%20R_0">
```
Comment (paulineribeyre) [inline generative_bayes_model/readme.md:33]: i didn't know you could do this in github markdown! cool!

File: generative_bayes_model/readme.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+![Alt text](images/cook_county_rt.svg?raw=true "Title")
+
+* Daily confirmed cases prediction (with 15-days forecast) in Cook County:
+
+![Alt text](images/cook_county_daily.svg?raw=true "Title")
+
+## Directory layout
+        .
+        ├── model_ealuation               # Direcory to evaluate the model prediction power
```
Comment (paulineribeyre) [inline generative_bayes_model/readme.md:54]:
```suggestion
        ├── model_evaluation               # Directory to evaluate the model prediction power
```

File: generative_bayes_model/readme.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        │   ├── model_evaluation.py       # Python scripts to evaluate model prediction power
+        │   ├── images                    # Exprected output for model evaluation
+        │   └── readme.md                 # How to use model_evaluation.py
+        ├── Dockerfile                    # Dockerfile for running generative model and model evaluation
```
Comment (paulineribeyre) [inline generative_bayes_model/readme.md:58]: `and model evaluation` => we don't run the model evaluation in the job, and we don't publish the evaluation chart on the portal, although we could if we wanted to

File: generative_bayes_model/model_evaluation/readme.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Evaluate the Model's Prediction Performance
+
+#### Fan Wang
+#### 12/08/2021
+-------------------
+## Objectives
+
+We use the prediciton model to forecast the future trend of the daily confirmed cases. It is important to establish a strong baseline of performance on the forecasting values. To this end, we need to relate the predicted values to the actual reported values. This python scripts implement the comparison between the predicted values and the true values. The users will have the flexibilities to choose the `start_date`, `end_date` and `prediction_window`, therefore the model performance could be evaluated with a suite of training and testing window sizes.
```
Comment (paulineribeyre) [inline generative_bayes_model/model_evaluation/readme.md:8]:
```suggestion
We use the prediction model to forecast the future trend of the daily confirmed cases. It is important to establish a strong baseline of performance on the forecasting values. To this end, we need to relate the predicted values to the actual reported values. This python scripts implement the comparison between the predicted values and the true values. The users will have the flexibilities to choose the `start_date`, `end_date` and `prediction_window`, therefore the model performance could be evaluated with a suite of training and testing window sizes.
```

File: generative_bayes_model/readme.md (hunk 3 above)
Comment (wangfan860) [reply generative_bayes_model/readme.md:58]: you are correct, we don't publish evaluation results to portal. But users may need the docker file to build their own image to run the evaluation code, and that's why I add `Dockerfile for running generative model and model evaluation`

File: generative_bayes_model/readme.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+![Alt text](images/cook_county_rt.svg?raw=true "Title")
+
+* Daily confirmed cases prediction (with 15-days forecast) in Cook County:
+
+![Alt text](images/cook_county_daily.svg?raw=true "Title")
+
+## Directory layout
+        .
+        ├── model_evaluation               # Directory to evaluate the model prediction power
```
Comment (paulineribeyre) [inline generative_bayes_model/readme.md:54]:
```suggestion
        ├── model_evaluation              # Directory to evaluate the model prediction power
```
woops, the extra character messed up your formatting

File: generative_bayes_model/readme.md (hunk 3 above)
Comment (paulineribeyre) [reply generative_bayes_model/readme.md:58]: i think it's not clear that the dockerfile in this directory doesn't do that

Comment (wangfan860) [reply generative_bayes_model/readme.md]: Got it. I just removed the `and model evaluation`.

**Review by paulineribeyre — DISMISSED** [review]

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**